apps/meteor/server/services/meteor/service.ts

```diff
@@ -146,14 +146,22 @@ export class MeteorService extends ServiceClassInternal implements IMeteor {
 			this.onEvent('watch.loginServiceConfiguration', ({ clientAction, id, data }) => {
 				if (clientAction === 'removed') {
 					serviceConfigCallbacks.forEach((callbacks) => {
-						callbacks.removed?.(id);
+						try {
+							callbacks.removed?.(id);
+						} catch (error) {
+							console.error(error);
+						}
```

Comment:
this might be a good opportunity to use `wrapExceptions`.. we can either `suppress` or `catch` and log
```suggestion
						wrapExceptions(() => callbacks.removed?.(id)).suppress();
```

Reply:
Done